task1/task1.Rmd

```diff
@@ -44,6 +44,43 @@ con <- dbConnect(drv, host='v45522.1blu.de', port=5432, user=getPass('Provide th
                  password=getPass('Provide the password'), dbname='datamanagement')
 ```
 
+```{r}
+# PREPARING THE DATA TO BE APPENDED TO THE QUALITY_CHECKED TABLE
+
+# load necessary packages
+library(tidyverse)
+library(lubridate)
+
+# create a list with all relevant HOBO IDs (there might be a more fancy way to do this)
+hobo_ids = c(10132405, 10234636, 10347312, 10347319, 10347346, 10347351, 10347357, 10347359, 10347364,
```

Comment:
I thought about 
hobo_ids = c(10132405 : 10801134)
but its actually (obviously) not working. So I think the way it is done originally is the proper way.

Reply:
I am not fully into this task, but couldnt you load the HOBO ids directly from the meta table in the Google sheet (URL is provided on the Exercise homepage).

Reply:
not all hobos got used and got quality checked. therefore, this would cause errors when trying to read something not existing in.

Reply:
ok, perhaps adding an additional column in the meta sheet and then filter HOBOs with the information given in this column...

Reply:
i think it's way more complicated. but if you think this works better and simpler, you can implement that. 😂👍

i thought more about something like reading in all filenames from the repository folder...